src/read.c

```diff
@@ -1500,41 +1517,58 @@ static avifResult avifDecoderGenerateImageGridTiles(avifDecoder * decoder, avifI
     return AVIF_RESULT_OK;
 }
 
-// Allocates the dstImage based on the grid image requirements. Also verifies some spec compliance rules for grids.
-static avifResult avifDecoderDataAllocateGridImagePlanes(avifDecoderData * data, const avifTileInfo * info, avifImage * dstImage)
+// Allocates the dstImage. Also verifies some spec compliance rules for grids, if relevant.
+static avifResult avifDecoderDataAllocateImagePlanes(avifDecoderData * data, const avifTileInfo * info, avifImage * dstImage)
```

Comment:
The changes to this function are an example of general cleanup/improvements that can be reviewed and checked in first.

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/2056, https://github.com/AOMediaCodec/libavif/pull/2057, https://github.com/AOMediaCodec/libavif/pull/2058, https://github.com/AOMediaCodec/libavif/pull/2059

Reply:
Thanks a lot!